spring-shell-core/src/main/java/org/springframework/shell/component/StringInput.java

```diff
@@ -26,11 +26,9 @@
 import org.jline.utils.AttributedString;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-
 import org.springframework.shell.component.StringInput.StringInputContext;
 import org.springframework.shell.component.context.ComponentContext;
 import org.springframework.shell.component.support.AbstractTextComponent;
-import org.springframework.shell.component.support.AbstractTextComponent.TextComponentContext;
```

Comment:
This is causing compile failure with jdk8

Reply:
Indeed, my IDE removed it as unused, reverting it